app/zpt/handler/atmos.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from scipy.integrate import odeint  # type: ignore
+from scipy.interpolate import BSpline, splrep  # type: ignore
+
+from .geos import g  # type:ignore
+
+AVOGADRO = 6.02282e23  # [mol^-1] aovogadros number
+
+
+def intermbar(z):
+    mbars = np.r_[
+        28.9644,
+        28.9151,
+        28.73,
+        28.40,
+        27.88,
+        27.27,
+        26.68,
+        26.20,
+        25.80,
+        25.44,
+        25.09,
+        24.75,
+        24.42,
+        24.10,
+    ]
+    mbarz = np.arange(85, 151, 5)
+    m = np.interp(z, mbarz, mbars)
+    return m
+
+
+def intatm(z, T, newz, normz, normrho, lat):
+    """Integrates the temperature profile to yield a new model
+    atmosphere in hydrostatic equilibrium including the effect of
+    g(z) and M(z).
+    newT, p, rho, nodens, n2, o2, o = intatm(z, T, newz, normz,normrho)
+    NOTE z in km and returns pressure in pa
+    """
+    wn2 = 0.78084  # mixing ratio N2
+    wo2 = 0.209476  # mixing ratio O2
+    k = 1.38054e-23  # jK-1 Boltzmans constant
```

Comment:
```suggestion
    k = 1.380649e-23 # jK-1 Boltzmans constant
```
This also has an exact definition: https://www.nist.gov/si-redefinition/meet-constants